app/models/driver.rb

```diff
@@ -0,0 +1,31 @@
+class Driver < ApplicationRecord
+  validates :name, presence: { message: "Please enter a name" }
+  validates :vin, presence:  {message: "Please enter a VIN" }
+
+  has_many :trips
+
+  def total_amount
+    total = 0
+
+    trips.each do |trip|
+      total += trip.cost
+    end
+    total -= (total *0.15)
+    return "$#{(total/100).round(2)}"
+  end
+
+  def average_rating
+    total = 0
+
+    trips.each do |trip|
```

Comment:
You could use an enumerable method to clean up this loop and simplify to a single line of code